### ISSUE TITLE
Add LinkedIn to K8s.io and K8s.dev

### ISSUE THREAD

**Issue by chris-short** [kubernetes/contributor-site#523]
Now that the community has control of the LinkedIn page, it should be added to the web sites.

/sig contribex

**Comment by sftim** [comment]
We should either track a similar issue for [k/website](https://github.com/kubernetes/website), or put a checklist on this one.

**Comment by Shubham82** [comment]
hi, JFI: The following similar issue is opened in k8s/website,
https://github.com/kubernetes/website/issues/46405

**Comment by sftim** [comment]
https://github.com/kubernetes/website/issues/46405 is broadly similar, but is about 𝕏 not LinkedIn.

**Comment by Shubham82** [comment]
yeah, you are right. I'm just wondering can we consider both icon 𝕏 and LinkedIn, while opening the the new issue or making a checklist on this issue?
@sftim your thoughts on this?

**Comment by sftim** [comment]
I'm happy for people to do this; I'd rather support / coach / review than lead on the implementation.

**Comment by sftim** [comment]
(SIG Docs current leadership can also help)

**Comment by chris-short** [comment]
Yes. I'm not cookie licking this. Anyone who isn't in the middle of a job search have time to take a look?

**Comment by chris-short** [comment]
We discussed this during today's Contributor Comms meeting and more in Slack after the meeting in the comms channel. At this point, we commit to taking X off both sites and have Bluesky, Mastodon, and LinkedIn accounts added to the footers. 

Personal note: Those are the places I'm paying more attention to personally and professionally, and there seems to be consensus with others in the community on that trio.

**Comment by chris-short** [comment]
#568 Fixes this

**Comment by kaslin** [comment]
We think this is dependent on fixes @SayakMukhopadhyay is making to k8s.dev. Also #568 which Chris mentioned, was closed but not merged.

**Comment by kaslin** [comment]
@SayakMukhopadhyay don't forget to update this if you have any updates!

**Comment by bupd** [comment]
/assign

I am happy to work on this.

**Comment by kaslin** [comment]
@bupd it looks like @TineoC already has the remaining work on this one coming.

/assign TineoC
